src/main/resources/application.yaml

```diff
@@ -153,11 +153,31 @@ job.scheduler:
 
 notification:
   english:
+    correctionRequest:
+      docmosisId: TB-SCS-LET-ENG-Action-Further-Evidence.docx
+    setAsideRequest:
+      docmosisId: TB-SCS-LET-ENG-Action-Further-Evidence.docx
+    libertyToApplyRequest:
+      docmosisId: TB-SCS-LET-ENG-Action-Further-Evidence.docx
+    sORRequest:
+      docmosisId: TB-SCS-LET-ENG-Action-Further-Evidence.docx
     jointPartyAdded:
       appellant:
         docmosisId: TB-SCS-GNO-ENG-00579.docx
       joint_party:
         docmosisId: TB-SCS-GNO-ENG-00579.docx
+    postHearingRequest:
+      docmosisId: TB-SCS-LET-ENG-representation-letter.docx
```

Comment:
Just confused about the indentation here. Want to confirm that this is intentional.

Reply:
removed this block because its not needed anymore